ci-operator/config/openstack-k8s-operators/openstack-k8s-operators-ci/openstack-k8s-operators-openstack-k8s-operators-ci-main.yaml

```diff
@@ -73,7 +73,7 @@ images:
   inputs:
     rhel-9-golang-1.21:
       as:
-      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.14
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.21-openshift-4.16
```

Comment:
I know that looks weird and confusing, but you should keep as `registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.14`, and the explanation is:
The build args is actually saying: in Dockerfile `images/build-root/Dockerfile.rhel-9.custom` [1] replace `registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.14` with `rhel-9-golang-1.21`.
And `rhel-9-golang-1.21` is defined above, in line 18.
[1] https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/blob/e45ea3a5485ce0e9ce024bff16e475f2fa60cd7c/images/build-root/Dockerfile.rhel-9.custom#L1C77-L1C86

just double cheking if image is correct:
```
podman run registry.ci.openshift.org/openstack-k8s-operators/tools:ci-build-root-golang-1.21-sdk-1.31 go version && operator-sdk version
go version go1.21.11 (Red Hat 1.21.11-1.el9_4) linux/amd64
operator-sdk version: "v1.31.0", commit: "e67da35ef4fff3e471a208904b2a142b27ae32b1", kubernetes version: "1.26.0", go version: "go1.19.11", GOOS: "linux", GOARCH: "amd64"
```

Reply:
reverted this change